test/test_jld2_output_writer.jl

```diff
@@ -266,11 +312,12 @@ for arch in archs
         test_field_slicing("sliced_funcs_jld2_test.jld2", ("u", "v", "w"), (4, 4, 4), (4, 4, 4), (4, 4, 5))
         test_field_slicing("sliced_func_fields_jld2_test.jld2", ("αt", "background_u"), (2, 4, 4), (2, 4, 4))
         
-        #####
-        ##### File splitting
-        #####
+        ####
+        #### File splitting
+        ####
 
-        test_jld2_file_splitting(arch)
+        test_jld2_file_splitting_size(arch)
+        test_jld2_file_splitting_time(arch)
```

Comment:
```suggestion
        test_jld2_size_file_splitting(arch)
        test_jld2_time_file_splitting(arch)
```

this doesn't matter, but does maintain our "english language style" where adjectives proceed nouns (yes, different than the way many codes are written, or French 😆 )